views/partials/banners/download_firefox_banner.hbs

```diff
@@ -7,7 +7,7 @@
       </div>
     </div>
     <div class="third">
-      <a href="https://www.mozilla.org/firefox/new/?utm_campaign=ffxmdownloads" class="button" target="_blank" rel="noopener">Download Firefox</a>
+      <a href="https://www.mozilla.org/firefox/new/?utm_source={{ UTM_SOURCE }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=download_banner" class="button" target="_blank" rel="noopener">Download Firefox</a>
```

Comment:
I think the only difference betweenst this and the partial below is that final `&utm_content=` value ("download_banner" versus "site-footer-link"). But I'm not convinced it's worth adding variables to the template to get this slightly DRYer.

```hbs
{{>email_button href=SERVER_URL value="Scan Another Email"}}
```
&mdash; via https://github.com/mozilla/blurts-server/pull/351/files#diff-aab49f213881484ad2be8406eddb6081R37

Reply:
Oh, good point. I'll file a follow-up TODO.